src/components/grid.css

```diff
@@ -1,68 +1,67 @@
 .outerWrapper {
-    display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(15em, 1fr));
-    grid-gap: 1rem;
-    grid-template-areas: 
-        "header"
-        "nav"
-        "page-content"
-        "footer";
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(15em, 1fr));
+  grid-gap: 1rem;
+  grid-template-areas: 
+    "header"
+    "nav"
+    "page-content"
+    "footer";
 }
 header {
-    grid-area: header;
+  grid-area: header;
 }
 nav {
-    grid-area: nav;
+  grid-area: nav;
 }
 main {
-    grid-area: page-content;
+  grid-area: page-content;
 }
 footer {
-    grid-area: footer;
+  grid-area: footer;
 }
 /* Media queries for grid layouts */
 @media screen and (min-width: 45em) {
-    .page-people > section {
-        display: grid;
-        grid-template-columns: repeat(2, 1fr);
-    }
-    .page-people > section > h1 {
-        grid-column: 1 / 3;
-    }
+  .page-people > section {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 1.5rem;
+  }
+  .page-people > section > h1 {
+    grid-column: 1 / 3;
+  }
 }
 @media screen and (min-width: 61em) {
-    header,
-    nav,
-    footer,
-    .page-home,
-    .page-people,
-    .page-person,
-    .page-news,
-    section.post {
-        display: grid;
-        grid-template-columns: repeat(12, 1fr);
-        gap: 1rem;
-    }
-    .logo,
-    .navbar,
-    .small-print,
-    .leader,
-    section.person,
-    .news,
-    section.post > .post-title,
-    section.post > .post-meta,
-    section.post > .post-content {
-        grid-column: 2 / 10;
-    }
-    .pagination {
-        grid-column: 2 / 12;
-    }
-    .page-people > section {
-        display: grid;
-        grid-column: 2 / 12;
-        grid-template-columns: repeat(3, 1fr);
-    }
-    .page-people > section > h1 {
-        grid-column: 1 / 4;
-    }
+  header,
+  nav,
+  footer,
+  #main-content > div,
+  section.post {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    grid-gap: 1rem;
+  }
+  .logo,
+  .navbar,
+  .leader,
+  .small-print,
+  section.person,
+  .news,
+  section.post > .post-title,
+  section.post > .post-meta,
+  section.post > .post-content {
+    grid-column: 2 / 12;
+  }
```

Comment:
This creates a main text block that is too wide. Not comfortable accepting this change.

Reply:
Also fair. Can we narrow the overall width, but still have it centered in the viewport. For example:

`grid-column: 3 / 11` 

This would make it the same width as before, just with even spacing on the left and right.

Reply:
Even 3 / 11 was ending up too wide. How about 3 / 10?

Reply:
In order to be centered on the page it has to have an even number of columns "padding" on the left and right. I like the width of 3 / 10, but that puts 2 columns on the left and 3 on the right, so it's not centered. 4 / 11 is too narrow, though, so perhaps we want to consider a different method of placing the site in the viewport? I had a simple `max-width` attribute on the wrapper class before, which I think would be all we need, combined with the existing mobile queries, which do a good job of making it appear nicely on smaller screens.

Reply:
I guess (perfectly) centered on the page doesn't feel critical to me.

I am not opposed to use `max-width`. The benefit of using the grid is that the grid positions are consistent across header/body/footer so it's easier to position things relative to each other if needed.